app/services/notices/setup/abstraction-alerts/fetch-monitoring-station.service.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+/**
+ * Orchestrates fetching the data needed for the Monitoring station journey
```

Comment:
I think we can just say...

```suggestion
 * Fetches the data needed for the monitoring station abstraction alert journey
```

Orchestration is more the purview of the calling service.